releasenotes/notes/fix_bqm_structured_error-98a72320a8759168.yaml

```diff
@@ -0,0 +1,7 @@
+---
+
+fixes:
+  - |
+    Fixes the ``bqm_structured`` decorator's error message for cases where
```

Comment:
This is misleading. I think it should say something like ` ``bqm_structured`` decorator now correctly raises a ``BinaryQuadraticModelStructureError`` even when there are no shared variables.` or something like that. It's not the message that's fixed but the error type.